dist/stat_data.py

```diff
@@ -528,6 +528,9 @@ def __init__(self, name, desc, flags=''):
     RecStat('rec_page_delete_fast', 'fast-path pages deleted'),
     RecStat('rec_pages', 'page reconciliation calls'),
     RecStat('rec_pages_eviction', 'page reconciliation calls for eviction'),
+    RecStat('rec_pages_with_prepare', 'page reconciliation calls that resulted in pages with prepared transaction metadata'),
+    RecStat('rec_pages_with_ts', 'page reconciliation calls that resulted in pages with timestamps'),
```

Comment:
I still feel like this is confusing personally. I would rather it say:
'page reconciliation calls that resulted in cells with timestamps', then the stat would be rec_pages_cells_with_ts. This would need to be consistent for all three stats.

Reply:
I feel using `rec_pages_cells_with_ts` and "resulted in cells with timestamps" is more confusing because it gives an impression that we are counting reconciled cells rather than the pages. Another important point to realize is that these stats are also often used by TSEs that have little knowledge of cell structure. 

@sulabhM - What do you think?

Reply:
Let’s use the phrase "resulted in values with timestamp" to indicate we are talking about cell values rather than the page address metadata.